### PR TITLE
Adding missed properties on IncrementalExecutionResultImpl.Builder.from

### PR DESCRIPTION
**Pull request by Juliano-Prado**
Adding two missed properties on IncrementalExecutionResultImpl.Builder.from

### CODE REVIEW

File: src/main/java/graphql/incremental/IncrementalExecutionResultImpl.java [hunk 1]
```diff
@@ -91,6 +91,8 @@ public Builder incrementalItemPublisher(Publisher<DelayedIncrementalPartialResul
         public Builder from(IncrementalExecutionResult incrementalExecutionResult) {
             super.from(incrementalExecutionResult);
             this.hasNext = incrementalExecutionResult.hasNext();
+            this.incremental = incrementalExecutionResult.getIncremental();
+            this.incrementalItemPublisher = incrementalExecutionResult.getIncrementalItemPublisher();
```
Comment (dondonz) [inline src/main/java/graphql/incremental/IncrementalExecutionResultImpl.java:95]: Perhaps you were already in the progress of doing it. Could you please add a test for this?

Comment (bbakerman) [reply src/main/java/graphql/incremental/IncrementalExecutionResultImpl.java:95]: Test please

Comment (Juliano-Prado) [reply src/main/java/graphql/incremental/IncrementalExecutionResultImpl.java:95]: I've added the tests, check out: src/test/groovy/graphql/incremental/IncrementalExecutionResultTest.groovy

File: src/test/groovy/graphql/incremental/IncrementalExecutionResultTest.groovy [hunk 2]
```diff
@@ -80,6 +83,41 @@ class IncrementalExecutionResultTest extends Specification {
...
+
+        def newIncrementalExecutionResult = new IncrementalExecutionResultImpl.Builder().from(incrementalExecutionResult).build()
 
+        then:
+        newIncrementalExecutionResult.incremental == incrementalExecutionResult.incremental
+        newIncrementalExecutionResult.extensions == incrementalExecutionResult.extensions
+        newIncrementalExecutionResult.errors == incrementalExecutionResult.errors
+        newIncrementalExecutionResult.incrementalItemPublisher == incrementalExecutionResult.incrementalItemPublisher
+        newIncrementalExecutionResult.hasNext() == incrementalExecutionResult.hasNext()
+        newIncrementalExecutionResult.toSpecification() == newIncrementalExecutionResult.toSpecification()
```
Comment (dondonz) [inline src/test/groovy/graphql/incremental/IncrementalExecutionResultTest.groovy:121]: I think this was a tiny typo, did you mean:

```suggestion
        newIncrementalExecutionResult.toSpecification() == incrementalExecutionResult.toSpecification()
```

**Review by bbakerman — APPROVED** [review]

**Pull request merged**